velox/expression/VectorFunction.h

```diff
@@ -137,6 +137,10 @@ class AlwaysFailingVectorFunction final : public VectorFunction {
   explicit AlwaysFailingVectorFunction(std::exception_ptr exceptionPtr)
       : exceptionPtr_{std::move(exceptionPtr)} {}
 
+  bool isDefaultNullBehavior() const override {
```

Comment:
Why this change? It doesn't look right.

Reply:
When the input arg is null, the exception in AlwaysFailingVectorFunction::apply won't be thrown if it's "true".